gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR for #184: Squad Requests

### Description

Implementing squad requests.

### Testing Directions

```
./setup
python3 manage.py runserver
```

1.  Login as vm1564@nyu.edu.
1.  Add vm1564+1@nyu.edu to your squad. Notice that they actually don't appear in your squad yet.
1.  Login as vm1564+1@nyu.edu and go to `/requests`. Notice that you have a request from vm1564@nyu.edu. Accept it OR Add vm1564@nyu.edu to your squad. Both actions will result in vm1564@nyu.edu and vm1564+1@nyu.edu merging squads.
1.  Repeat the above with squads of size 2 or more. Ensure that the CSS on `/requests` looks good. Test the "Deny" button (the row in the Request table will be deleted, allowing another request to be sent).

### Miscellaneous Comments

*   We don't have a notification mechanism that notifies the user that a request was sent. Should file an issue for it.

### Checklist
